tests/datasets/dataset_test.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Unit tests for //compiler_gym/datasets."""
+from pathlib import Path
+
+import pytest
+
+from compiler_gym.datasets.dataset import Dataset
+from tests.test_main import main
+
+pytest_plugins = ["tests.pytest_plugins.common"]
+
+# pylint: disable=abstract-method
+
+
+def test_dataset_properties(tmpwd: Path):
+    del tmpwd  # Unused
```

Comment:
why is the argument required for the test function?

Reply:
Good catch! In an early revision, `Dataset` would create a directory on init, so the test needed a scrap working directory to write to. This is no longer required, I'll remove it.